console/backend/src/main/java/org/frankframework/management/web/Init.java

```diff
@@ -21,149 +21,118 @@
 import java.util.List;
 import java.util.Map;
 
-import jakarta.ws.rs.DELETE;
-import jakarta.ws.rs.DefaultValue;
-import jakarta.ws.rs.GET;
-import jakarta.ws.rs.POST;
-import jakarta.ws.rs.PUT;
-import jakarta.ws.rs.Path;
-import jakarta.ws.rs.Produces;
-import jakarta.ws.rs.QueryParam;
-import jakarta.ws.rs.core.MediaType;
-import jakarta.ws.rs.core.Response;
-
 import jakarta.annotation.security.PermitAll;
 import jakarta.annotation.security.RolesAllowed;
-import org.apache.cxf.jaxrs.model.ClassResourceInfo;
-import org.apache.cxf.jaxrs.model.MethodDispatcher;
-import org.apache.cxf.jaxrs.model.OperationResourceInfo;
-
-/**
- * Root collection for API.
- *
- * @since	7.0-B1
- * @author	Niels Meijer
- */
-
-@Path("/")
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.method.HandlerMethod;
+import org.springframework.web.servlet.mvc.method.RequestMappingInfo;
+import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
+import org.springframework.web.util.pattern.PathPattern;
+
+@RestController
 public class Init extends FrankApiBase {
 
+	@Autowired
+	private RequestMappingHandlerMapping handlerMapping;
+
 	private boolean isMonitoringEnabled() {
 		return getProperty("monitoring.enabled", false);
 	}
 
-	@GET
+	@GetMapping(value = {"", "/"}, produces = "application/json")
```

Comment:
Why do we have `""` as input value? 
I guess only `@GetMapping("/")` should be sufficient here already.

Reply:
`/api/endpoint` vs `/api/endpoint/`

Reply:
Ah clear! So, it should be both then. Ok.